shared_vcx/src/errors/error.rs

```diff
@@ -53,3 +53,85 @@ where
 }
 
 pub type SharedVcxResult<T> = Result<T, SharedVcxError>;
+
+pub mod prelude {
+    pub use crate::errors::error::*;
+}
+
+#[derive(Copy, Clone, Eq, PartialEq, Debug, thiserror::Error)]
+pub enum SharedAgencyClientErrorKind {
```

Comment:
There already is `SharedVcxErrorKind` under `shared_vcx/src/errors/error.rs`, just extend that one please

Reply:
Please also check if all of these added error kinds are actually being used.